### PR TITLE
New features and better error handling 

### PR DESCRIPTION
**Pull request by prachil007**
1. Added recent configurations
2. Added connection pool status 
3. Added export options for queries
4. Handled some crucial errors

### CODE REVIEW

**Comment by prachil007** [comment]
Closing in favor of some corrections

File: readme.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
 
-The queries that retrive primary keys for table are hardcoded for postgres, so if you want to use it for any other db, you will need to change those queries.
+The queries that retrieve primary keys for table are hard-coded for PostgreSQL, so if you want to use it for any other DB, you will need to change those queries.
 
+## IMPORTANT NOTES
 
-You can use json instead of typing in all the input fields
+### Important note for recent configurations
+
+- While making jar executable, make sure to add write permissions. Use 'chmod +xw ./sysconfigurator.jar'
```
Comment (kvineet) [inline readme.md:14]: You want write permission on the directory not jar file.

File: src/main/java/io/kvineet/sysconfigurator/utils/StringUtils.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package io.kvineet.sysconfigurator.utils;
+
+public class StringUtils {
+
+	public static boolean isNullOrEmpty(String str) {
```
Comment (kvineet) [inline src/main/java/io/kvineet/sysconfigurator/utils/StringUtils.java:5]: just use apache commons StringUtils.

**Review by kvineet — COMMENTED** [review]
Good work

**Pull request closed without merging**